tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -2616,6 +2620,17 @@ describe('scalar card', () => {
           end: null,
         });
       }));
+
+      it('toggles selected time when single fob is deselected in step selector', fakeAsync(() => {
```

Comment:
maybe: toggles step selector when single fob is deselected in step selector

Reply:
removed "step selector" cause this is nested in the "step selector feature integration" test